ldid.cpp

```diff
@@ -2268,6 +2269,10 @@ Hash Sign(const void *idata, size_t isize, std::streambuf &output, const std::st
         get(common, name, NID_commonName);
     }
 
+    if (flag_t != NULL) {
+        printf("ldid: Using team ID %s\n", flag_t);
```

Comment:
Maybe don’t print this. No other flag prints stuff when it’s in use.

Reply:
Yep, looks like I forgot to remove this from testing